modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -1,36 +1,35 @@
-The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+To create a binding connection with a service that is backed by the Crunchy Postgres for Kubernetes, add the Crunchy Postgres for Kubernetes Operator to the *OperatorHub*, and install the Operator. The Crunchy Postgres for Kubernetes Operator provides the PostgresCluster resource that you can use to deploy a PostgreSQL database and expose the binding information in secrets, config maps, status, and spec attributes.
 
 .Prerequisites
 
-* You have created and deployed a Spring PetClinic sample application by using the *Developer* perspective.
-* You have installed {servicebinding-title} from the OperatorHub.
-* You have installed the *Crunchy Postgres for Kubernetes* Operator from the OperatorHub by using the `v5` *Update* channel.
-* You have created and deployed a Crunchy PostgreSQL database instance by using the *Developer* perspective. This instance has the following components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and `hippo-pgbouncer`.
+* You created and deployed a Spring PetClinic sample application in the *Developer* perspective.
+* You installed {servicebinding-title} from the *OperatorHub*.
+* You installed the *Crunchy Postgres for Kubernetes* Operator from the OperatorHub in the `v5` *Update* channel.
+* You created a *PostgresCluster* resource in the *Developer* perspective, which resulted in a Crunchy PostgreSQL database instance with the following components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and `hippo-pgbouncer`.
 
 .Procedure
 
-. Switch to the *Developer* perspective and ensure that you are in the appropriate project, for example, `my-petclinic`. 
+. In the *Developer* perspective, switch to the relevant project, for example, `my-petclinic`.
 . In the *Topology* view, hover over the Spring PetClinic sample application to see a dangling arrow on the node.
-. Click and drag the arrow towards the *hippo* database in the Postgres Cluster to make a binding connection with the Spring PetClinic sample application. 
+. Drag and drop the arrow onto the *hippo* database icon in the Postgres Cluster to make a binding connection with the Spring PetClinic sample application.
 
-.. Enter the name and click *Create*.
+. In the *Create Service Binding* dialog, keep the default name or add a different name for the service binding, and then click *Create*.
 +
 .Service Binding dialog
 image::odc-sbc-modal.png[]
-
-
-Alternatively, in the *+Add* view, click the *YAML* option to see the *Import YAML* screen. Use the YAML editor and add the `ServiceBinding` resource:
-
+. Optional: If there is difficulty in making a binding connection using the Topology view, go to *+Add* -> *YAML* -> *Import YAML*.
+. Optional: In the YAML editor, add the `ServiceBinding` resource:
```

Comment:
You could actually get this of these two lines. See how I turned them into a tip in my suggestion in https://github.com/openshift/openshift-docs/pull/49201/files#r975819799.

Reply:
@max-cx @Dhruv-Soni11 This alternative flow is important within the procedure to aid the user in completing the task. I do not prefer moving it elsewhere.

Reply:
Keeping it as is.